website/docs/users-sources/sources/social-logins/google/workspace/index.md

```diff
@@ -0,0 +1,205 @@
+
+Organizations using Google Workspace allow their users to authenticate into applications using their company email addresses. This guide shows how to set up Security Assertion Markup Language (<abbr>SAML</abbr>) as the authentication method between Google Workspace and authentik.
+
+## SAML Authentication Flow
+
+This sequence diagram shows a high-level flow between user, authentik, Google Workspace, and the target application.
+
+```mermaid
+sequenceDiagram
+    autonumber
+    participant User
+    participant authentik
+    participant Google
+    participant App
+
+    User->>App: User navigates to app...
+    App-->>authentik: ← Redirected to authentik
+    Note over authentik: User chooses "Google Workspace"
+    authentik-->>Google: Redirect to Google →
+    Note over Google: Authenticate with email and password
+    Google-->>authentik: ← Redirect to authentik
+    authentik-->>App: Redirect to app →
+```
+
+In short, the user navigates to the application, is redirected to authentik, chooses Google Workspace as the authentication method, authenticates with Google, and is redirected back to the application.
+
+The key characteristic that makes this an IdP-to-IdP flow is that authentik is acting as an intermediary identity provider, brokering trust between your application and Google Workspace.
+
+---
+
+## Preparation
+
+By the end of this integration, your authentik instance will allow users to authenticate using their Google Workspace credentials.
+
+You'll need to have authentik instance running and accessible on an HTTPS domain, and a Google Workspace domain with super-administrator access.
+
+Keep a text-editor handy because we'll be copying and pasting values between the two services.
+
+### Placeholders
+
+The following placeholders are used:
+
+- `authentik.company`: The Fully Qualified Domain Name of the authentik installation.
+
+## Google Workspace configuration
+
+We'll need some information from Google to complete the integration, so
+start by logging into [Workspace Admin Console](https://admin.google.com/) as a super-admin.
+
+### Create a new application
+
+From the Workspace Admin Console, navigate to the _Apps_ section, and then to _Web and mobile apps_.
+Continue by expanding the **Add app** dropdown and selecting **Add custom SAML app.**
+
+Within the app creation page, define the following **Name** and **Description** for the new application.
+
+| Field       | Value                        |
+| ----------- | ---------------------------- |
+| Name        | authentik                    |
+| Description | Single Sign-On for authentik |
+
+Press **Continue** to generate the SAML configuration we'll need to complete the integration.
+
+### Google Identity Provider details
+
+You should now be presented with a choice to download metadata file containing the SAML configuration, or copy the details to your clipboard.
+
+Under _Option 2_, copy the SSO URL to your text editor and download the certificate.
```

Comment:
It's still not clear where exactly this Option 2 is... name of page, where on which page is it, how do they find it...?

Reply:
I feel you. I’m also a bit puzzled on how to convey this step without making it more complex. The page is split into two sections titled “Option 1” and “Option 2”. The first section only contains a download button for the metadata. The second section titled “Option 2” has a series of labeled input fields to copy from. 

Sadly, we can’t change Google’s copy but I think the reader will be able to find their way. 😊

Reply:
Thanks for looking at it again. Does the overall page have a name? Even that bit of guidance would be useful. Or, this might be a rare place where we do want a screenshot!

Reply:
The page header aligns with our title: "Google Identity Provider details"

IMO I'm hoping to avoid screenshots because they get stale so quickly. Given that this a subsection of docs under "Google Workspace configuration", I believe that the user will be able to follow the flow. This step is also one of the easier tasks since they're only copying information provided by Google, rather than filling out more fields.

Reply:
When writing instructions, procedurals, I def state the place where they are to do the task. However, this doc is written in a more friendly, narrative form of instructions... so sure, lets leave as is.